src/routes/feeds/index.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Request, Response } from 'express';
+import { createClient } from 'redis';
+import axios from 'axios';
+import transactionWrapper from '../transactionWrapper';
+import authMiddleware from '../../middleware/authMiddleware';
+import { generateRedisConfig } from '../../lib/queue';
+
+const {
+  uri: redisUrl,
+  tlsEnabled,
+} = generateRedisConfig();
+
+const feedUrl = 'https://acf-ohs.atlassian.net/wiki/createrssfeed.action?types=page&pageSubTypes=comment&spaces=conf_all&title=Confluence+RSS+Feed&labelString=releasenotes&excludedSpaceKeys%3D&sort=modified&maxResults=10&timeSpan=5&showContent=true&confirm=Create+RSS+Feed&os_authType=basic';
+
+const getFeeds = async (_req: Request, res: Response) => {
+  const redisClient = createClient({
```

Comment:
I think it makes sense to abstract this redis call into something like 
```js
getCachedResponse(key, callback)
```
so we can use it for other things, like the resource dashboard for example

Reply:
Do you want to do that as part of this PR or wait until its needed in more places?

Reply:
I'm waiting to actually see how they are using the RSS feed to do anymore work on this, but I expect that to be fairly soon.

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1407/files